### ISSUE TITLE
Does it also supports Windows with Anaconda? 

### ISSUE THREAD

**Issue by ghimiredhikura** [traveller59/second.pytorch#113]
Hello @traveller59, 

I think you haven't yet tested this project in Windows with Anaconda support. 
Do you have any plan to make it work in Windows too? 

Thanks.

**Comment by ghimiredhikura** [comment]
Yes, I also had difficulty with spconv library. If it can be built in windows other things seems to be straightforward.

**Comment by traveller59** [comment]
I have success build and import the spconv in windows. I will add windows support for spconv soon.

**Comment by traveller59** [comment]
I have tested train/eval in windows 10/cuda 10. The only requirement of windows support is spconv which is solved in newest commit. 
The major problem of windows is speed, the spconv is much slower on windows than ubuntu, I will try but I can't promise to solve this.

**Comment by ghimiredhikura** [comment]
Thanks for your effort. How much speed degraded in windows as compared to ubuntu. Can you provide some numbers.

**Comment by traveller59** [comment]
evaluation in car.lite.config in 1060 notebook:
Windows 10:
```
generate label finished(27.22/s). start eval:
avg example to torch time: 1.558 ms
avg prep time: 5.970 ms
avg voxel_feature_extractor time = 0.523 ms
avg middle forward time = 11.897 ms
avg rpn forward time = 9.267 ms
avg predict time = 6.493 ms
```
Linux: 
generate label finished(38.31/s). start eval:
avg example to torch time: 0.697 ms
avg prep time: 5.022 ms
avg voxel_feature_extractor time = 0.230 ms
avg middle forward time = 7.392 ms
avg rpn forward time = 9.494 ms
avg predict time = 2.718 ms

This may due to lots of cuda kernel launchs. currently implementation require 3x3x3x5=135 custom kernel launch (not include torch::_unique and torch::mm) in one forward for 3x3x3 kernel.

**Comment by Robert-Hopkins** [comment]
how to install spconv on Windows？thank you
